lib/mastodon/domains_cli.rb

```diff
@@ -157,6 +157,48 @@ def crawl(start = nil)
       end
     end
 
+    option :url, type: :string, default: '', aliases: [:u]
+    desc 'import [URL]', 'Import blocked domains based on the csv url provided'
+    long_desc <<-LONG_DESC
+      Imports all the block list in the form of csv response
+      There are two ways to import a csv, the exported pattern of the csv
+      and the domain only way of pattern
+    LONG_DESC
+    def import_blocked(url)
+      domains_csv = HTTP.get(url).body.readpartial
+      default_headers = %w(#domain #severity #reject_media #reject_reports #public_comment #obfuscate)
+
+      domains = CSV.parse(domains_csv, headers: default_headers)
+      created = 0
+      merged = 0
+
+      domains.each do |row|
+        domain = row['#domain'].strip
+
+        if DomainBlock.rule_for(domain).present?
+          say("#{domain} is already present in the block list or is ruled out", :red)
```

Comment:
Did you mean to increment `merged` here?

Reply:
Created https://github.com/magicstone-dev/ecko/pull/272 to address this.